javascript/src/interfaces/profile.ts

```diff
@@ -0,0 +1,21 @@
+//profie List
+export interface ProfileListItem {
```

Comment:
Please do not export **`ProfileListItem`**. Instead, use and import **`ProfileListResponseData`**, in the **ControllerProfile** and **index**.

 ```suggestion
interface ProfileListItem {
```